core/http/Cargo.toml

```diff
@@ -24,13 +24,19 @@ hyper = { version = "0.10.13", default-features = false }
 time = "0.1"
 indexmap = "1.0"
 rustls = { version = "0.13", optional = true }
+dns-lookup = "0.9.1"
+untrusted = "0.6.1"
+webpki = "0.18.0-alpha4"
+openssl = "0.10.10"
```

Comment:
Rocket simply can't depend on `openssl` because of these issues. Whatever is being used here from `openssl` can almost certainly be found in `ring`, `rustls`, or another `ring` based library. We should use one of those instead.

Reply:
Is it just the `openssl` dependency that is a problem, or are there other dependencies you don't like here?